tensornetwork/backends/tensorflow/tensorflow_backend.py

```diff
@@ -430,3 +430,18 @@ def eps(self, dtype: Type[np.number]) -> float:
       float: Machine epsilon.
     """
     return tf.experimental.numpy.finfo(dtype).eps
+
+  def pinv(self, tensor: Tensor, rcond: float = 1E-15, hermitian: bool = False) -> Tensor:
+    """
+    Compute the (Moore-Penrose) pseudo-inverse of a tensor.
+    Returns the pseudo-inverse of tensor.
+    
+    Args:
+     tensor: A tensor.
+     rcond: Cutoff for small singular values.
+     hermitian(optional): If True, matrix provided is assumed to be Hermitian (symmetric if real-valued). Defaults to False.
+
+    Returns:
+     tensor: The pseudo inverse of tensor.
+         """
+    return np.linalg.pinv(tensor)
```

Comment:
you should use the tensorflow version here

Reply:
Hi.
In a [StackOverflow article](https://stackoverflow.com/questions/60025950/tensorflow-pseudo-inverse-doesnt-work-for-complex-matrices/60128892#60128892) someone has mentioned that the Tensorflow version doesn't work for complex matrices and that's why I went with the Numpy Version.
So shall I still use the Tensorflow version only?

Reply:
yes, please use tensorflow. The point here is to use backend-native operations.